miniapps/solvers/plor_solvers.cpp

```diff
@@ -59,10 +59,10 @@
 //    mpirun -np 4 plor_solvers -m ../../data/star-surf.mesh -fe r
 //
 // Device sample runs:
-//    mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe h -d cuda
-//    mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe n -d cuda
-//    mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe r -d cuda
-//    mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe l -d cuda
+//  * mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe h -d cuda
+//  * mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe n -d cuda
+//  * mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe r -d cuda
+//  * mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe l -d cuda
```

Comment:
In the nightly tests, only the following sample run has produced non-deterministic results:
```console
mpirun -np 4 plor_solvers -m ../../data/fichera.mesh -fe r -d cuda
```
Maybe we can enable the other 3?

Reply:
We discussed this and decided to keep all 4 sample runs disabled.